tests/FSharp.Compiler.ComponentTests/Conformance/TypesAndTypeConstraints/IWSAMsAndSRTPs/IWSAMsAndSRTPsTests.fs

```diff
@@ -320,6 +320,33 @@ module ``Equivalence of properties and getters`` =
             IL_000e:  ret
           }"""]
 
+module ``Type checking behavior`` =   
+
+    #if !NETCOREAPP
+    [<Theory(Skip = "IWSAMs are not supported by NET472.")>]
```

Comment:
A minor comment - we shouldn't probably put the source in the inlinedata, since it'll end up in the test case name.
It's better to make the language version an inline data and keep the source code in the body.

Reply:
Thanks - I added a helper function to the `Compiler.fs` test utility that accepts a version string.